readme.md

```diff
@@ -173,6 +169,14 @@ Once you do a search, you should see a list similar to the one below (versions m
 Please use [GitHub Issues](https://github.com/windows-toolkit/WindowsCommunityToolkit/issues) for bug reports and feature requests.
 For general questions and support, please use [Stack Overflow](https://stackoverflow.com/questions/tagged/windows-community-toolkit) where questions should be tagged with the tag `windows-community-toolkit`.
 
+## <a name="supported"></a> Required Dependencies
+The following dependencies are required for building the Windows Community Toolkit repo and sample app:
+
+* [Visual Studio 2019](https://visualstudio.microsoft.com/downloads/)
+* [Windows SDK May 2019 Update	18362](https://developer.microsoft.com/en-us/windows/downloads/sdk-archive)
+* [.NET Core 3.0.100 SDK](https://dotnet.microsoft.com/download/visual-studio-sdks)
```

Comment:
I have tested with .NET Core 3.0.**103** SDK, It's working as expected. I think we should remove the specific version number.
```suggestion
* [.NET Core 3.0 SDK](https://dotnet.microsoft.com/download/visual-studio-sdks)
```

Reply:
Thanks Vijay, now that @azchohfi has done PR #3164 we can just put 3.1 here, I'll update this PR and then I think we can get this merged in too now.,